packages/algob/src/lib/deployer.ts

```diff
@@ -85,13 +89,14 @@ export class AlgobDeployerImpl implements AlgobDeployer {
     return this.cpData.precedingCP[this.networkName].asa[name];
   }
 
-  async deployASC (name: string, source: string, account: Account): Promise<ASCInfo> {
+  async deployASC (name: string, scParams: Object, flags: ASCDeploymentFlags): Promise<ASCInfo> {
```

Comment:
scParams should have a type, don't leave `Object` here.

Reply:
scParams will be defined by user, how do i classify it in a type ?

Reply:
Ok. Then it's fine.